### PR TITLE
Add header and headline section for mobile

### PR DESCRIPTION
**Pull request by alonsofl**
### Summary
Added the header with the menu and the headline section with a short description and social icons for mobile.
Note: Following the [Template #3](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468).

#### Changes
- Configured linters for the project (HTML & CSS).
- Updated README.md file.
- Added .gitignore file (to ignore node_modules folder).
- Added global styles for margin, padding, box-sizing, and font-family in main.css.
- Added color variables for the project.
- Added header with "Welcome" message and hamburger menu icon.
- Added headline section with a short description and supporting text, using the colors for the project.
- Added a background image for the section.
- Added social icons for "Twitter", "LinkedIn" and "GitHub".
- Added a down arrow icon link to navigate to the next section.

### CODE REVIEW

**Review by Shaher-11 — COMMENTED** [review]
## _Status:_ Invalid:x:
## Hello @alonsofl 
This Code Review Request will be marked as invalid in your Dashboard because you didn't follow the correct [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) as required for this project.
Please make the necessary changes and submit another code review request once you are ready! :smiley:

File: index.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles/main.css">
+  <title>Alonso Falconi</title>
+</head>
+<body>
+  <header class="main-menu">
+    <h1 class="menu-title">Welcome</h1>
```
Comment (mricanho) [inline index.html:12]: - [ ] Welcome it's not a title, here works kind of a logo or a subtitle, so change it for an "a" tag or a "hx"

File: index.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    <button class="btn menu-btn" type="button">
+      <img src="images/menu.svg" alt="Menu">
+    </button>
+  </header>
+  <main>
+    <section class="headline">
+      <h2 class="headline-title">Hey there. I'm Alonso<br><span class="headline-description">I'm a software developer</span></h2>
```
Comment (mricanho) [inline index.html:19]: - [ ] This is actually the main title, so, treat it as an "h1" tag please

File: index.html [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+      <p class="headline-support-text">I can help you build a product, feature, or website. Look through some of my work and experience! If you like what you see and have a project you need to be coded, don't hesitate to contact me.</p>
+      <div class="social-icons">
+        <a href="https://www.twitter.com/alonsofalconi"><img src="images/twitter.svg" alt="Twitter"></a>
+        <a href="https://www.linkedin.com/in/alonsofalconi"><img src="images/linkedin.svg" alt="LinkedIn"></a>
+        <a href="https://www.github.com/alonsofl"><img src="images/github.svg" alt="GitHub"></a>
+      </div>
```
Comment (mricanho) [inline index.html:25]: - [ ] The list of icons uses a `ul` tag to group them, and `li` per icon; each icon must have a link (`a` tag)

File: README.md [hunk 4]
```diff
@@ -1 +1,18 @@
-# dev-portfolio
\ No newline at end of file
+# My Portfolio Project
+
+Project to create a portfolio website.
```
Comment (mricanho) [inline README.md:3]: - [ ] [OPTIONAL] You can be a little bit more descriptive with this part of the README file

**Review by mricanho — CHANGES_REQUESTED** [review]
Hi @alonsofl, 

Good job so far! 🔥🔥

It is a very good first attempt Alonso, you are following the gitflow, your commit messages are descriptive, and you are using semantic HTML super well, congrats.
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

almost there

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @alonsofl!

You worked hard in all the instructions I gave you before👍🏽 

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**